gif-screencast.el

```diff
@@ -121,13 +121,83 @@ various programs run here."
   :init-value nil
   :keymap gif-screencast-mode-map)
 
+(defvar gif-screencast--done nil
+  "Flag if capturing already done.")
+
+(defvar gif-screencast--finishing nil
+  "Flag if compilation in progress.")
+
+(defvar gif-screencast--counter 0
+  "Number of running screenshots.")
+
+(defun gif-screencast-capture-sentinel (_proc _status)
+  "Sentinel for screen capturing."
+  (setq gif-screencast--counter (- gif-screencast--counter 1))
+  (gif-screencast--finish))
+
+(defun gif-screencast-done ()
```

Comment:
Why making this a function?  It's a one-liner and called only once.

Reply:
Only for readability. I can remove it if you want.

Reply:
Yes, I think it's not worth a separate function.